com.unity.multiplayer.mlapi/Runtime/NetworkVariable/Collections/NetworkSet.cs

```diff
@@ -331,29 +331,6 @@ IEnumerator IEnumerable.GetEnumerator()
             return m_Set.GetEnumerator();
         }
 
-        /// <inheritdoc />
```

Comment:
Git has made a mess of this diff; I just hoisted the Add functionality out of the interface-qualified definitions into a method and then made those interfaces call that method.